examples/02-Merlin-Models-and-NVTabular-applying-to-your-own-dataset.ipynb

```diff
@@ -0,0 +1,698 @@
+{
```

Comment:
<p>"calculations" sounds odd. Maybe "operations" or "transformations"?</p><p> </p><p>"Categorify userId and movieId, that the values are continuous integers from 0 ... |C|" - what's |C| in this context, as there are two categorical variables with different cardinalities</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/253/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>